uisdk/src/main/res/values/styles.xml

```diff
@@ -310,15 +337,16 @@
     </style>
 
     <style name="KhBookingTermsText">
-        <item name="android:background">@drawable/uisdk_background_light_grey_rounded</item>
         <item name="android:textSize">@dimen/text_size_small</item>
         <item name="android:textColor">@color/colorAccent</item>
     </style>
 
     <style name="KhBookingTermsTitleText">
-        <item name="android:background">@drawable/uisdk_background_light_grey_rounded</item>
         <item name="android:textSize">@dimen/text_size_small</item>
-        <item name="android:textColor">@color/text_alternative</item>
+        <item name="android:textColor">@color/black</item>
+        <item name="android:fontFamily">@font/barlow_regular</item>
+        <item name="fontFamily">@font/barlow_regular</item>
+        <item name="fontWeight">600</item>
```

Comment:
can we define a palette for different fontWeights? (if it is needed after removing the custom font)

```
<dimen name="kh_uisdk_font_weight_normal">400</dimen>
<dimen name="kh_uisdk_font_weight_bold">500</dimen>
<dimen name="kh_uisdk_font_weight_bolder">600</dimen>
```

Reply:
Since we've removed the custom fonts, they're not needed in the uisdk anymore... at least for now